rpc-protocol/src/main/java/org/sonarsource/sonarlint/core/rpc/protocol/backend/connection/auth/HelpGenerateUserTokenParams.java

```diff
@@ -20,14 +20,59 @@
 package org.sonarsource.sonarlint.core.rpc.protocol.backend.connection.auth;
 
 
+import javax.annotation.CheckForNull;
+import javax.annotation.Nullable;
+
 public class HelpGenerateUserTokenParams {
   private final String serverUrl;
 
+  private final Utm utm;
+
   public HelpGenerateUserTokenParams(String serverUrl) {
+    this(serverUrl, null);
```

Comment:
Is the goal that all clients implement the "feature"? If so, you could deprecate this constructor